pages/establishment/update/content/fields.js

```diff
@@ -0,0 +1,26 @@
+module.exports = {
+  name: {
+    label: 'Establishment name'
+  },
+  address: {
+    label: 'Address'
+  },
+  licences: {
+    label: 'Licenced to carry out',
+    options: {
+      supplying: 'Supply of relevant protected animals',
+      breeding: 'Breeding  of relevant protected animals',
+      procedure: 'Regulated scientific procedures on regulated animals'
+    }
+  },
+  authorisations: {
+    label: 'Authorisations',
+    options: {
+      killing: 'Methods of killing not specified in Schedule 1',
+      rehomes: 'Setting free and rehoming of protected animals after completion of a series of regulated procedures'
+    }
+  },
+  comments: {
+    label: 'Why are you making this change?'
+  }
+};
```

Comment:
A lot fo the content in this file feels like it's being repeated, and possibly should be shared in a common file.